README.md

```diff
@@ -58,6 +58,15 @@ Frameworks to the Functions Framework contract.
           -type=legacyevent \
           -buildpacks=false
         ```
+    - **Ruby __typed http__** function Example:
+
+        ```sh
+        $HOME/functions-framework-conformance/client/client \
+          -cmd="bundle exec functions-framework-ruby --source test/conformance/app.rb --target typed_func --signature-type http" \
+          --type=http \
+          --declarative-type=typed \
+          -buildpacks=false
+        ```
```

Comment:
Lets update [the usage table](https://github.com/GoogleCloudPlatform/functions-framework-conformance/blob/master/README.md#usage) and step 1 of the [quick start] (https://github.com/GoogleCloudPlatform/functions-framework-conformance/blob/master/README.md#quickstart)